frontend/conf/DEV.public.conf

```diff
@@ -21,11 +21,11 @@ google.oauth {
 }
 
 membership {
-    url = "https://mem.thegulocal.com"
+    url = "https://mem.thegulocal.com:9100"
```

Comment:
Ok, I'll remove them. This is one of the many pitfalls I encountered while trying to run this thing end-to-end locally.